<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetConta</class>
 <widget class="QWidget" name="WidgetConta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_20">
     <item row="0" column="1">
      <widget class="QLabel" name="labelContasPagar">
       <property name="text">
        <string>Contas a pagar</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="TableView" name="tableContasPagar">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditBuscaContasPagar">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelContasPagarBuscar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Buscar:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBox_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Filtros:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButtonContaPagarPago">
          <property name="text">
           <string>Pagos</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButtonContaPagarPendente">
          <property name="text">
           <string>Pendentes</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>290</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButtonContaPagarLimpar">
          <property name="text">
           <string>Todos</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Filtros:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButtonContaReceberPendente">
          <property name="text">
           <string>Pendentes</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>290</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButtonContaReceberRecebido">
          <property name="text">
           <string>Recebidos</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButtonContaReceberLimpar">
          <property name="text">
           <string>Todos</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelContasReceber">
       <property name="text">
        <string>Contas a receber</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="TableView" name="tableContasReceber">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelContasReceberBuscar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Buscar:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditBuscaContasReceber">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
